common/src/main/java/jp/co/soramitsu/common/compose/component/Utils.kt

```diff
@@ -26,3 +30,9 @@ fun Image(modifier: Modifier = Modifier, @DrawableRes res: Int, contentDescripti
         contentDescription = contentDescription
     )
 }
+
+fun Modifier.clickableWithNoIndication(onClick: () -> Unit): Modifier {
```

Comment:
I've put it in there: common/src/main/java/jp/co/soramitsu/common/utils/ComposeExt.kt we should leave one